iFixit/react-commerce
# Add page template lifetime warranty section

closes #1556 

❗❗❗After this PR is merged Strapi should be redeployed in production ❗❗❗

cc @sterlinghirsh @danielbeardsley @masonmcelvain 

## QA

1. Open [store home page preview](https://react-commerce-git-add-page-template-lifetime-war-ac49ac-ifixit.vercel.app/store)
2. Verify that lifetime warranty section is rendered
3. Verify that the section is responsive
4. Verify that you can change the section content from [Strapi](https://add-page-template-lifetime-warranty-section.govinor.com/admin/content-manager/collectionType/api::page.page/1?plugins[i18n][locale]=en)
